js/main.js

```diff
@@ -118,6 +131,37 @@ function setStartup(lStartup){
     });
 }
 
+// Method to overwrite user config on mac installer
+function updateUserConfigMac() {
+    return new Promise((resolve, reject) => {
+        let userConfigPath = dirs.userConfig() + '/';
+        let globalConfigPath = process.argv[2];
+        let userName = process.env.USER;
+
+        childProcess.exec(`rsync -r "${globalConfigPath}" "${userConfigPath}" && chown -R "${userName}" "${userConfigPath}"`, {timeout: 60000}, (err) => {
+            if (err) {
+                reject(err);
+            }
+            resolve();
+        });
+    });
+}
+
+// Method to overwrite user config on windows installer
+function updateUserConfigWin() {
```

Comment:
We should use a common method for mac and windows. Just is the isMac flag to check and take action accordingly.